### ISSUE TITLE
Make wasm_exec.js copy from go sources in Makefile

### ISSUE THREAD

**Issue by joeyak** [zorchenhimer/MovieNight#97]
This is an issue because the wasm_exe.js file breaks versions when a 1.13 version used with 1.12 and vice versa.

**Comment by zorchenhimer** [comment]
The workaround for this is gonna be pretty hacky.  Getting the path to the `wasm_exe.js` from the currently installed version of Go requires modifying the output of `go env GOTOOLDIR`.

The problem arises on Windows because of its use of backslashes instead of forward slashes in paths.  This breaks the Makefile when you try to use that value pretty much anywhere as it expects only forward slashes in paths.

I *think* I have a working solution, but it isn't all that elegant.  I've basically got a "script" in the folder that the `wasm_exec.js` is expected to be that copies the file over.  This "script" is invoked with `go run static/js/GetWasmExec.go` and it seems to work.  I want to test it a bit more before committing it.

Thoughts?

**Comment by crossworth** [comment]
I was about to create an issue about this as well.

On *unix we can use the following code without much trouble.
```
updateWasm:
	cp $(shell go env GOROOT)/misc/wasm/wasm_exec.js static/js/wasm_exec.js
```

On windows under Cygwin/Cmder it works too, but I'm not sure about cmd or powershell.


Another approach would be using `go generate` to copy the file.

**Comment by joeyak** [comment]
So, I would like to avoid having a script to copy the file. I'll look into the go generate. I like that idea. Don't like the idea of removing comparability with cmd or powershell

**Comment by crossworth** [comment]
To be honest for me cmd and powershell are already broken because of the env vars

```
GOOS=js GOARCH=wasm  go build ...
```

I know there is a cross environment way of setting env vars with `go env`, something like:
```
go env -w GOOS=js GOARCH=wasm 
```

but I never looked into it.

**Comment by zorchenhimer** [comment]
This has been fixed with #108
